### ISSUE TITLE
Lights not responding anymore

### ISSUE THREAD

**Issue by oricalworks** [nikshriv/cync_lights#71]
my Cync dimmer switches do not respond to HA anymore... pls help

**Comment by dougkl** [comment]
Same issue here

**Comment by theandrewlane** [comment]
Also experiencing this 🤦🏻‍♂️

**Comment by Menz01** [comment]
Same here

**Comment by brandon-daniels** [comment]
Mine stopped working after I updated to HA Core 2023.7.2. I rolled back the update and it started working again. I suggest everyone does the same until a fix is released.

**Comment by Menz01** [comment]
> Mine stopped working after I updated to HA Core 2023.7.2. I think it might be the updated integrations page (Core 2023.6). I rolled back the update (to Core 2023.2) and it started working again.

rolled all the way back to the February release?

**Comment by brandon-daniels** [comment]
> > Mine stopped working after I updated to HA Core 2023.7.2. I think it might be the updated integrations page (Core 2023.6). I rolled back the update (to Core 2023.2) and it started working again.
> 
> rolled all the way back to the February release?
Yes, lol. I am slow about updates because of situations like this.
